src/platform/forms/tests/save-in-progress/RoutedSavableApp.unit.spec.jsx

```diff
@@ -432,6 +421,5 @@ describe('Schemaform <RoutedSavableApp>', () => {
 
     expect(fetchInProgressForm.called).to.be.false;
     expect(router.replace.calledWith('/first-in-form-page')).to.be.true;
-    __BUILDTYPE__ = buildType;
```

Comment:
These `BUILDTYPE` mutations were no longer necessary, because the module being tested only has special behavior on the `LOCALHOST` environment, and unit tests run under `VAGOVDEV`.